### ISSUE TITLE
reduce image size in test folder

### ISSUE THREAD

**Issue by ebgoldstein** [UNCG-DAISY/psi-collect#86]
>Try reducing the size data in the tests folder. Do you really need to include the full resolution images to test on? Or is it possible to use a reduced size image or even mock them out completely. This might improve compatibility with build systems (which can have issues if the repo is too big) and might just generally be easier to work with.

from @chrisleaman 's JOSS review: https://github.com/openjournals/joss-reviews/issues/2075#issuecomment-584910432

**Comment by Matmorcat** [comment]
Yes. Some of the earlier tests were fairly inefficient, and I'd like to rewrite many of them as mocking out the actual accessing of web content and files would make the tests much more robust and reliable. For the current tests, the images can actually just be near-empty files, so long as the tests that check file sizes are updated as well.
